### ISSUE TITLE
Add a link to the TRACE eval website

### ISSUE THREAD

**Issue by mister-walter** [khoury-gsa/khoury-gsa.github.io#6]
I can do this later. Just a placeholder for now.

https://www.applyweb.com/eval/shibboleth/neu/36892

**Comment by ulysses4ever** [comment]
Good idea! I wonder what would be a good place on the page for it. Probably somewhere in the Useful Resources section. But we shouldn't bike-shed it much.
